client/src/Router.elm

```diff
@@ -24,9 +24,9 @@ import Url exposing (Protocol(..), Url)
 type Page
     = Login
     | Catalogue
-    | AddLevel1
-    | AddLevel0
-    | AddBackbone
+    | AddLevel1Page
+    | AddLevel0Page
+    | AddBackbonePage
```

Comment:
Why? This change makes the naming inconsistent. I strongly recommend undoing this change

Reply:
Main.elm already has those as type Msg. I did it to avoid confusion.

Reply:
Ah ok. Fair enough. Can you make the naming consistent too then: `LoginPage`, `CataloguePage`, and `AdminPage`..

Reply:
Done!